{{ cookiecutter.project_name }}/requirements.txt

```diff
@@ -1 +1,3 @@
 prefect
+pytest
+python-dotenv
```

Comment:
Die wil ik niet. Environment variabelen horen via het environment te worden doorgegeven door docker-compose (en later eventueel kubernetes).
Het moet niet afhankelijk worden van een lokaal bestandje.
(Nou gaat docker compose misschien wel een lokaal bestandje gebruiken, maar ik verwacht dat dat via config in prefect kan?)